components/inputs/test/input-time-range.test.js

```diff
@@ -163,33 +162,38 @@ describe('d2l-input-time-range', () => {
 				{ enforceTimeIntervals: false, validStart: false, validEnd: false }
 			].forEach((testCase) => {
 				it(`when enforceTimeIntervals = ${testCase.enforceTimeIntervals}, valid start = ${testCase.validStart} and valid end = ${testCase.validEnd}`, async() => {
+
+					const defaultStartValue = formatDateInISOTime(getDefaultTime(undefined, testCase.enforceTimeIntervals, 'ten'));
+					const defaultEndTime = getDateFromISOTime(defaultStartValue);
+					defaultEndTime.setMinutes(defaultEndTime.getMinutes() + 10);
+					const defaultEndValue = formatDateInISOTime(defaultEndTime);
```

Comment:
So sometime between when these tests were skipped because they were timing out in legacy-Edge and now, we changed the behaviour of the code. It now uses the current time when coming up with the default value, not midnight.